src/muya/lib/contentState/updateCtrl.js

```diff
@@ -2,6 +2,27 @@ import { tokenizer } from '../parser/'
 import { conflict } from '../utils'
 import { CLASS_OR_ID } from '../config'
 
+const getParentCheckBox = function (checkbox) {
+  const parent = checkbox.parentElement.parentElement.parentElement
```

Comment:
I think we can put `getParentCheckBox `, `cumputeChecboxStatus ` function in utils folder. and put `setCheckBoxState `, `updateParentsCheckBoxState `, `updateChildrenCheckBoxState `, `listItemCheckBoxClick ` methods in clickCtrl.js file.